hack/jenkins/build.sh

```diff
@@ -62,7 +62,7 @@ git clean -fdx
 # docker images -q | xargs -r docker rmi
 
 # Build
-go run ./hack/e2e.go -v --build
+make release
```

Comment:
Oh dear, that's been changed since I last looked over that code. I guess I should add a check for `KUBE_FASTBUILD`.